examples/notebooks/beam-ml/automatic_model_refresh.ipynb

```diff
@@ -132,8 +119,6 @@
         "from typing import Tuple\n",
         "\n",
         "import apache_beam as beam\n",
-        "from apache_beam.examples.inference.tensorflow_imagenet_segmentation import PostProcessor\n",
-        "from apache_beam.examples.inference.tensorflow_imagenet_segmentation import read_image\n",
```

Comment:
We still need to inline this function somewhere (probably alongside the mapping function where we call it)

Reply:
Can you explain it? I didn't understand `inline` this function?

Reply:
Right now, read_image is still called, but its not defined anywhere. Originally we imported it, but that's fragile since the example could change/doesn't have rigorous API guarantees. We should define the function next to its usage instead

Reply:
Yeah, we use `preprocess_image` and this change was also lost in the commit. I am running the notebook with updated changes.